packages/widget-playground/src/components/DrawerControls/DesignControls/ColorControls.tsx

```diff
@@ -44,11 +71,11 @@ const ColorSelector = ({
 
   return colorValue ? (
     <ColorSelectorContainer {...rest}>
-      {colorName}
+      <CapitalizeFirstLetter>{colorName}</CapitalizeFirstLetter>
       <ColorInput
         aria-label={`${colorName} color selection`}
         type="color"
-        value={colorValue}
+        value={safe6DigitHexColor(colorValue)}
```

Comment:
The browsers color input doesn't like 3 digit hex colors. This does a safe conversion if they are featured

Reply:
Maybe it is better to agree on using 6-digit values in the themes?

Reply:
Maybe moving foward thats a good idea. If we ever get to making the config updatable we will likely have to think about doing things like this as well